conductr_cli/conduct_load_license.py

```diff
@@ -0,0 +1,31 @@
+from conductr_cli import license, validation
+from conductr_cli.constants import DEFAULT_LICENSE_FILE
+from conductr_cli.exceptions import LicenseLoadError
+import logging
+
+
+@validation.handle_connection_error
+@validation.handle_http_error
+@validation.handle_license_load_error
+def load_license(args):
+    log = logging.getLogger(__name__)
+
+    license_file = DEFAULT_LICENSE_FILE
+
+    if args.offline_mode:
+        log.info('Skipping downloading license from Lightbend.com')
```

Comment:
What if we are in offline mode and the license file is not on disk yet. Then we should abort.